g3/config/handler.py

```diff
@@ -0,0 +1,88 @@
+import configparser
+from pathlib import Path
+from typing import Optional
+
+CONFIG_DIR = ".g3"
+CONFIG_FILE = "config"
+
+
+class Handler:
+    def __init__(self, config_path: Path = Path.home()):
+        self.config_path = config_path / CONFIG_DIR
+        self.config_file = self.config_path / CONFIG_FILE
+        self.config = self.load_config()
+        self.properties = self.load_properties()
+
+    @property
+    def as_dict(self) -> dict[str, any]:
+        return self.properties
+
+    @property
+    def github_token(self) -> str:
+        return self.properties.get("github_token")
+
+    @property
+    def openai_key(self) -> str:
+        return self.properties.get("openai_key")
+
+    @property
+    def api_base(self) -> Optional[str]:
+        return self.properties.get("api_base")
+
+    @property
+    def deployment_id(self) -> Optional[str]:
+        return self.properties.get("deployment_id")
+
+    @property
+    def api_type(self) -> Optional[str]:
+        return self.properties.get("api_type")
+
+    @property
+    def model(self) -> str:
+        return self.properties.get("model")
+
+    @property
+    def temperature(self) -> Optional[float]:
+        return self.properties.get("temperature")
```

Comment:
I think that `properties.get("key")` will return a `str | None`. So for the case of `temperature` we might need to convert it to a float, either here or at the `OpenAIConfig` class.

Reply:
Thanks